test/integration/template_database_test.go

```diff
@@ -3221,3 +3221,52 @@ func TestDatabaseStatefulSetVolumeSnapshot(t *testing.T) {
 		}
 	})
 }
+func TestDatabaseAdminAffinityLabels(t *testing.T) {
+	// Path to the helm chart we will test
+	helmChartPath := testlib.DATABASE_HELM_CHART_PATH
+
+	options := &helm.Options{
+		SetValues: map[string]string{
+			"admin.affinityLabels":           "zone host",
+			"database.sm.noHotCopy.replicas": "1",
+			"database.sm.hotCopy.replicas":   "1",
+		},
+	}
+
+	// Run RenderTemplate to render the template and capture the output.
+	output := helm.RenderTemplate(t, options, helmChartPath, "release-name", []string{"templates/statefulset.yaml"})
+
+	for _, obj := range testlib.SplitAndRenderStatefulSet(t, output, 2) {
+		require.NotEmpty(t, obj.Spec.Template.Spec.Containers)
+
+		container := obj.Spec.Template.Spec.Containers[0]
+
+		found := false
+		for i, arg := range container.Args {
+			if arg == "--admin-affinity-label-keys" {
+				assert.Equal(t, "zone host", container.Args[i+1])
+				found = true
+			}
+		}
+
+		assert.True(t, found)
+	}
+
+	output = helm.RenderTemplate(t, options, helmChartPath, "release-name", []string{"templates/deployment.yaml"})
```

Comment:
Where is `helmChartPath` variable set to `testlib.ADMIN_HELM_CHART_PATH`?

Reply:
Line 3226:
`helmChartPath := testlib.DATABASE_HELM_CHART_PATH`

This test is looking at the TE start arguments, so I am using the database helm chart, not the admin. Am I wrong?

Reply:
Please disregard my comment.